bloom-housing/bloom
# fix: unit tables multiple rows for same rent type, sorting issues

# Pull Request Template

## Issue

Addresses HBA#247 #1297 

## Description

This PR tackles two issues. The first is that when I made the change to show multiple rows for different rent types (% income vs static number), it also ended up showing multiple rows for the same type if the values were different. For example, Coliseum appeared as below with the bug. There should be two rows for 2 BR, not three. The different static rent numbers should appear as a range.
<img width="493" alt="Screen Shot 2021-06-01 at 5 31 23 PM" src="https://user-images.githubusercontent.com/65367387/120403567-aaa11880-c301-11eb-819b-67725975dd79.png">

With this fix, it now appears as below.
<img width="755" alt="Screen Shot 2021-06-01 at 5 31 31 PM" src="https://user-images.githubusercontent.com/65367387/120403630-cc9a9b00-c301-11eb-809b-17c6b8a4cc4a.png">

The second issue this solves is unit summaries coming through unsorted (see the linked issues).

I also just tried my best to clean up the fxns, they were confusing.

## Type of change


## How Can This Be Tested/Reviewed?

Import a listing with rent ranges, like [Coliseum](https://github.com/housingbayarea/bloom/blob/0.3_alameda/backend/core/listing_data/coliseum.json). Check out the unit summaries table, the occupancy ranges, and the floor ranges.

## Checklist:
